wordpress-mobile/WordPress-Android
# AppBar refresh - Me, Editor, Site Creation, Media Picker and Site Preview

This PR updates AppBar design and adds lift on scroll functionality to the following screens (and related nested screens)

- Me screen and all the nested screens (excluding Zendesk).
- Editor screens - Post Settings and Revisions.
- Site Creation flow (only the first step)
- Media Picker.
- Site Preview.

@osullivanchris Unfortunately I was not able to get a lift on scroll working in editor :( Any ideas on what we can do there instead?

To test:

For all screens:

- Make sure nothing looks out of place (including ActionMode) :)
- Make sure the toolbar state is restored on configuration change.

Test 1:

- Navigate to the Me screen and confirm that lift on scroll works (you might need to check the screen in landscape orientation).
- Check My Profile,  Account Settings, App Settings, and Help screens and confirm that lift on scroll works.

Test 2:

- Navigate to the Editor screen and check Post Settings and History + History Details screens to confirm that lift on scroll works.

Test 3:

- Start Site Creation flow and confirm that lift on scroll works on the first step.

Test 4:

- Open media picker (from the editor or by tapping on Blavatar on My Site screen) and confirm that lift on scroll works.

Test 4:

- Open Site Preview and confirm that lift on scroll works.



PR submission checklist:
